README.md

```diff
@@ -9,17 +9,35 @@ storing them locally, and on a schedule send them out to the Automattic
 servers. Realistically this library is only useful for Automattic-based
 projects but the idea is to share what we've made.
 
-## Build
-
-* Build:
```

Comment:
I don't think this section has much added value to be part of our `README`, so I wanted to suggest of its removal. This is because `./gradlew assemble` is one of the most basic Gradle tasks and works for all projects.